index.html

```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
+                </li>
+            </ul>
+                <button class="button-4">See Project</button>
+          
+        </div>
+        
+        </div>
+    </section>
+    <section>
+        <div class="container-1">
+            <a href="#" class="project-pics"><img class="project-pics" src="./images/Snapshoot Portfolio (3).png" alt=""></a>
+           <div class="flow">
+            <h3 class="project-title-1">Multi-Post <br> Stories</h3>
+            <ul class="info">
+                <li class="conopy-li">
+                    <h3 class="canopy">CANOPY</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li class="back-end-li">
+                    <h3 class="back-end">Back End Dev</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li>
+                    <h3 class="year">2015</h3>
+                </li>
+            </ul> 
+        <p class="info-about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="info-2">
+                <li>
+                    <button class="button-1">html</button>
+                        
+                </li>
+                <li>
+                    <button class="button-2">css</button>
+                        
+                    
+                </li>
+                <li>
+                    <button class="button-3">Javascript</button>
+                        
+                    
+                </li>
+            </ul>
+                <button class="button-4">See Project</button>
           
         </div>
-    
+        
+        </div>
+    </section>
+    <section>
+        <div class="about-my-self">
+        <div class="about-container">
+            <h1 class="about-title">About <br> Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h2>LET’S CONNECT</h2>
+            <ul class="media-icons1">
+                <li>
+                    <a href=""><img src="./images/github.svg" alt=""></a>
+                </li>
+                <li>
+                    <a href=""><img src="./images/Linkedin.svg" alt=""></a>
+                </li>
+                <li>
+                    <a href=""><img src="./images/v.svg" alt=""></a>
+                </li>
+                <li>
+                    <a href=""><img src="./images/twiter.svg" alt=""></a>
+                </li>
+                <li>
+                    <a href=""><img src="./images/m.png" alt=""></a>
+                </li> 
+            </ul>
+            <Button>Get my resume</BUtton>
+        </div>
```

Comment:
I suggest you add equal `paddings and margins` to your project. Currently the left side has more one while the right does not. Its part of the requirements that your project matches figma design
<img width="1660" alt="Screenshot 2022-10-29 at 10 36 44" src="https://user-images.githubusercontent.com/48546992/198821326-5f17ec4e-7652-4a64-a5a4-8d2f57ef9449.png">

this is your design:
<img width="366" alt="Screenshot 2022-10-29 at 11 12 27" src="https://user-images.githubusercontent.com/48546992/198821339-b7dfe9d0-e560-4df9-a100-9fa3994be122.png">

This is the figma design
<img width="373" alt="Screenshot 2022-10-29 at 11 12 03" src="https://user-images.githubusercontent.com/48546992/198821350-3486a6b4-ccf5-4e5d-855f-87711426d7d7.png">